docs/guide/variadics/README.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Variadic Nodes
+---
+
+# Variadic Nodes
+
+There are operations which can be applied to an arbitrary number of input
+values. For example, consider the addition operation. Sure, intuitively you
+know how it works for two values, three values, or 42 values:
+
+* 1 + 6 = 7
+* 1 + 6 + 3 = 10
+* 1 + 6 + 3 + …more summands… = someNumber
+
+Other examples of such functions include number multiplication, string
+concatenation or joining, booleans AND’ing or OR’ing, n-th element choice, and
+so on.
+
+To natively support such operations, XOD offers a mechanism called *variadic*
+nodes. The variadic nodes can scale the number of their input pins to accept
+more or fewer values.
+
+In XOD IDE the variadic nodes have a little gripper on the right, and the pins
+which can scale are marked with the ellipsis (⋯) on mouse hover. Depending on
+requirements you can add groups of variadic pins up to an infinite number. Drag
+the right border of the node to adjust.
+
+![Overview](./overview.patch.png)
+
+The variadic pins replication count is called *arity level*. By default, the
```

Comment:
"The variadic pins replication count" looks a bit scary, but I can't come up with a better alternative. @brusherru, what do you think?

Reply:
The number of variadic pin copies is called arity level?

Reply:
> The number of variadic pin copies is called arity level?

I like the original variant more :)